### ISSUE TITLE
 What is the accuracy of each neural net?

### ISSUE THREAD

**Issue by appleeji** [Xilinx/CHaiDNN#29]
I checked the deploy.prototxt of the neuralnet provided in the demo file, and found that the LRN layer was removed. This would lead to a greater accuracy decline than the actual Ristretto claims of 1% accuracy.
However, CHaidnn's documentation does not specify how accurate the accuracy is.
Actually, when I run chaidnn on the zynQ board, I see that it shows almost similar results, but how accurate is it?
I would like to know how much of a drop in accuracy compared to the actual caffe model
plus, Are you planning to add an LRN layer in the future?

**Comment by smenon009** [comment]
Hi @appleeji 
Accuracy details can be found here : https://github.com/Xilinx/CHaiDNN/blob/master/docs/MODELZOO.md   

Yes, removing LRN layer and using the same old model will lead to bad accuracy. It will require some cycles of re-training/ finetuning to re-gain the lost accuracy. CHai V2 enables users to add some custom layer implementations. Please check the software layer plugin feature here: https://github.com/Xilinx/CHaiDNN/blob/master/docs/SOFTWARE_LAYER_PLUGIN.md

**Comment by VishalX** [comment]
Closing.
